BackofficeBundle/Resources/public/less/gridstack.less

```diff
@@ -0,0 +1,28 @@
+#template {
+    .jarviswidget, div[role="content"], widget-body, .grid-stack{
+        background-color: #92a2a8;
+        background-clip: content-box;
+    }
+}
+.grid-stack-item-content {
+    background: white none repeat scroll 0 0;
+    color: #92a2a8;
+    border-radius: 10px;
+    font-size: 20px;
+    text-align: center;
+}
+.grid-stack-item-content .fa {
+    display: block;
+    font-size: 64px;
+    margin: 20px 0 10px;
+}
+header a, header a:hover {
+    color: #fff;
+}
+.darklue {
+    background: #2c3e50 none repeat scroll 0 0;
+    color: #fff;
+}
+.darklue hr.star-light::after {
+    background-color: #2c3e50;
+}
```

Comment:
Are those css an override of the grid stack css, or are they actually the original ones ?